### PR TITLE
Prevent connecting an AudioNode to an AudioParam of a different AudioContext

### LINKED ISSUE #1126: Connecting an AudioNode to an AudioParam of a different BaseAudioContext is not explicitely forbidden
We need to state that it throws.

### PR DESCRIPTION
**Pull request by padenot**
This fixes #1126.

### CODE REVIEW

**Comment by rtoy** [comment]
Weird. I can't see them either.

File: index.html [hunk 1]
```diff
@@ -2560,7 +2560,12 @@ <h2 id="OfflineAudioCompletionEvent">
                 The <code>destination</code> parameter is the
                 <a><code>AudioParam</code></a> to connect to. This method does
                 not return <code>destination</code>
-                <a><code>AudioParam</code></a> object.
+                <a><code>AudioParam</code></a> object. <span class=
+                "synchronous">If <var>destination</var> belongs to an
```
Comment (rtoy) [inline index.html]: Is "belongs" correct, spec-wise?

File: index.html [hunk 2]
```diff
@@ -2560,7 +2560,12 @@ <h2 id="OfflineAudioCompletionEvent">
...
+                <a>AudioNode</a> that belong to a <a>BaseAudioContext</a> that
```
Comment (rtoy) [inline index.html]: s/belong/belongs/

File: index.html (hunk 1 above)
Comment (padenot) [inline index.html]: I'm defining this properly in a branch I'm doing to define the AudioNode ctors and other ctors, but I agree this is a bit lose for now.

Comment (rtoy) [inline index.html]: Do you want to land this now or fix it up later in your upcoming ctor branch?

Comment (padenot) [inline index.html]: I'll land now, thanks.

**Pull request merged**